Frechet_distance/test/Frechet_distance/Frechet_distance_test.cpp

```diff
@@ -0,0 +1,344 @@
+    using Point = Kernel::Point_2;
+    std::cout <<"CGAL::Epick\n";
+    double t1=Test_struct<Kernel, Traits, Point>::testFrechetDistanceNearNeighborsDS();
+    double t2=Test_struct<Kernel, Traits, Point>::testFrechetDistance();
+    std::cout << t1 << " " << t2 << "\n";
+  }
+
+  if (test_set.empty() || test_set.count(2))
+  {
+    using Kernel = CGAL::Epeck;
+    using Traits = CGAL::Frechet_distance_traits_2<Kernel>;
+    using Point = Kernel::Point_2;
+    std::cout <<"CGAL::Epeck\n";
+    double t1=Test_struct<Kernel, Traits, Point>::testFrechetDistanceNearNeighborsDS();
+    double t2=Test_struct<Kernel, Traits, Point>::testFrechetDistance();
+    std::cout << t1 << " " << t2 << "\n";
+  }
+
+  if (test_set.empty() || test_set.count(3))
+  {
+    using Kernel = CGAL::Exact_predicates_exact_constructions_kernel_with_sqrt;
+    using Traits = CGAL::Frechet_distance_traits_2<Kernel>;
+    using Point = Kernel::Point_2;
+    std::cout <<"Exact_predicates_exact_constructions_kernel_with_sqrt (force filtering)\n";
+    double t1=Test_struct<Kernel, Traits, Point>::testFrechetDistanceNearNeighborsDS();
+    double t2=Test_struct<Kernel, Traits, Point>::testFrechetDistance<true>();
+    std::cout << t1 << " " << t2 << "\n";
+  }
+
+  if (test_set.empty() || test_set.count(4))
+  {
+    using Kernel = CGAL::Simple_cartesian<CGAL::Exact_rational>;
+    using Traits = CGAL::Frechet_distance_traits_2<Kernel>;
+    using Point = Kernel::Point_2;
+    std::cout <<"Simple_cartesian<Exact_rational> (force filtering)\n";
+    double t1=Test_struct<Kernel, Traits, Point>::testFrechetDistanceNearNeighborsDS();
+    double t2=Test_struct<Kernel, Traits, Point>::testFrechetDistance<true>();
+    std::cout << t1 << " " << t2 << "\n";
+  }
+
+  if (test_set.empty() || test_set.count(5))
+  {
+    using Kernel = CGAL::Simple_cartesian<double>;
+    using Traits = CGAL::Frechet_distance_traits_3<Kernel>;
+    using Point = Kernel::Point_3;
+    std::cout <<"Simple_cartesian<double> in 3D\n";
+    double t1=Test_struct<Kernel, Traits, Point>::testFrechetDistance();
+    std::cout << t1 << "\n";
+  }
+
+  if (test_set.empty() || test_set.count(6))
+  {
+    using Kernel = CGAL::Simple_cartesian<double>;
+    using Traits = CGAL::Frechet_distance_traits_3<Kernel>;
+    using Point = Kernel::Point_3;
+    std::cout <<"Simple_cartesian<double> in 3D (force filtering)\n";
+    double t1=Test_struct<Kernel, Traits, Point>::testFrechetDistance<true>();
+    std::cout << t1 << "\n";
+  }
+
+  if (test_set.empty() || test_set.count(7))
+  {
+    using Kernel = CGAL::Epick_d<CGAL::Dimension_tag<100>>;
+    using Traits = CGAL::Frechet_distance_traits_d<Kernel>;
+    using Point = Kernel::Point_d;
+    std::cout <<"CGAL::Epick_d\n";
+    double t1=Test_struct<Kernel, Traits, Point>::testFrechetDistance();
+    std::cout << t1 << "\n";
+  }
+
+  if (test_set.empty() || test_set.count(8))
+  {
+    using Kernel = CGAL::Epeck_d<CGAL::Dimension_tag<100>>;
```

Comment:
If I replace 100 with 3, it fails to compile (interaction between `Bbox_3` and `Bbox_d<Dimension_tag<3>>`).
Curiously, the Epick_d version does compile with dim 3, although it gives an (expected) assertion failure at runtime because the file format differs.

Reply:
This is now fixed.